src/NewCommand.php

```diff
@@ -40,10 +41,36 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+
         if (! extension_loaded('zip')) {
             throw new RuntimeException('The Zip PHP extension is not installed. Please install it and try again.');
         }
 
+        $presets_path = getenv('HOME').DIRECTORY_SEPARATOR."laravel-installer.json";
```

Comment:
`$HOME` is usually not set in Windows by default.

Reply:
Cool, i'll look into this - might be a use case for the posix functions.

Reply:
Possibly this: https://stackoverflow.com/a/21473475/941446